case-lib/relay.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2021-2025 Intel Corporation. All rights reserved.
+
+# test-mic_privacy.sh needs to control mic privacy settings (on/off)
+# needs usbrelay package: https://github.com/darrylb123/usbrelay
+# param1: switch name
+# param2: switch state
+usbrelay_switch()
+{
+    # Declare a constant for the relay settle time
+    USBRELAY_SETTLE_TIME=0.5
+
+    local switch_name=$1
+    local state=$2
+
+    dlogi "Setting usbrelay switch $switch_name to $state."
+    usbrelay "$switch_name=$state" > /dev/null || {
+        # if not detect relays hw module, skip the test
+        die "Failed to set usbrelay switch $switch_name to $state.
+        The usbrelay hw module is not responding or no relays detected.
+        Check hardware connection."
+    }
+
+    # wait for the switch to settle
+    sleep "$USBRELAY_SETTLE_TIME"
+
+    # Display current state of the switch
+    current_state=$(usbrelay | grep "$switch_name" | awk -F= '{print $2}')
```

Comment:
`awk` has `grep` built-in.
```suggestion
    current_state=$(usbrelay | grep  awk -F= "/$switch_name/ {print \$2}")
```

Or, if you prefer
```suggestion
    current_state=$(usbrelay | awk -F= "/$switch_name/"' {print $2}')
```